core/packages/test/scripts/set-env.sh

```diff
@@ -19,6 +19,7 @@ relay_bin="$relay_dir/build/snowbridge-relay"
 [ "$bridge_hub_runtime" = "bridge-hub-rococo-local" ] &&
    seconds_per_slot=6 ||
    seconds_per_slot=12
+active_spec="${ACTIVE_SPEC:-minimal}"
```

Comment:
Lets please add a comment explaining this field. We need to start improve the documentation for this script.